OpenSim/Common/TimeSeriesTable.h

```diff
@@ -106,6 +106,10 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
     TimeSeriesTable_& operator=(const TimeSeriesTable_&) = default;
     TimeSeriesTable_& operator=(TimeSeriesTable_&&)      = default;
     ~TimeSeriesTable_()                                  = default;
+
+    using DataTable_<double, ETY>::DataTable_;
+    using DataTable_<double, ETY>::operator=;
+    using DataTable_<double, ETY>::flatten;
```

Comment:
Wow I thought this only works for constructors! Could you maybe add a comment describing what these do? Especially since people will be flattening TimeSeriesTables most of the time.